src/main/java/org/broadinstitute/hellbender/tools/walkers/annotator/DepthPerAlleleBySample.java

```diff
@@ -58,7 +59,9 @@ public void annotate(final ReferenceContext ref,
         for ( final Allele allele : vc.getAlleles() ) {
             alleleCounts.put(allele, 0);
         }
-        likelihoods.bestAlleles(g.getSampleName()).stream()
+        final Map<Allele, List<Allele>> alleleSubset = alleles.stream().collect(Collectors.toMap(a -> a, a -> Arrays.asList(a)));
```

Comment:
You could use a method reference here: `Collectors.toMap(a -> a, Arrays::asList)`. I'm not saying it's better, just wanted to mention that that's an option too.